src/app/clusters/network-commissioning/network-commissioning.cpp

```diff
@@ -135,6 +136,24 @@ void Instance::Shutdown()
     mpBaseDriver->Shutdown();
 }
 
+#if !CHIP_DEVICE_CONFIG_SUPPORTS_CONCURRENT_CONNECTION
+void Instance::SendNonConcurrentConnectNetworkResponse()
+{
+    auto commandHandleRef = std::move(mAsyncCommandHandle);
+    auto commandHandle    = commandHandleRef.Get();
+    if (commandHandle == nullptr)
+    {
+        return;
+    }
+
+    DeviceLayer::ConnectivityMgr().GetBleLayer()->mState = Ble::BleLayer::kState_Disconnecting;
```

Comment:
This needs to be conditional on CONFIG_NETWORK_LAYER_BLE

Reply:
Done